containers/compose/nginx/nginx.conf

```diff
@@ -0,0 +1,90 @@
+error_log /dev/stdout info;
```

Comment:
We have to keep this file in sync with something else.

Is it roles/pulp-web/templates/pulp-web.configmap.yaml.j2 or something from another repo?

Reply:
I added it more as example, the idea is having the user changing everything under compose dir, as it is a shared volume